boards/xtensa/lilygo_t3/Kconfig.defconfig

```diff
@@ -0,0 +1,21 @@
+# Lilygo T3 board configuration
+
+# Copyright (c) 2024 Lothar Felten <lothar.felten@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD
+	default "lilygo_t3"
+	depends on BOARD_LILYGO_T3
+
+config ENTROPY_GENERATOR
+	default y
+
+config HEAP_MEM_POOL_ADD_SIZE_BOARD
+	int
+	default 98304 if WIFI
+	default 40960 if BT
+	default 4096
```

Comment:
this has recently been modified -- please look how ESP32 boards define this in `main`.

Reply:
Made the same change as Sylvio Alves did to other boards